stac_pydantic/api/search.py

```diff
@@ -111,6 +111,9 @@ def validate_datetime(cls, v):
             if value == "..":
                 dates.append(value)
                 continue
+            elif value == "":
+                dates.append("..")
+                continue
```

Comment:
I think this might be clearer as:

```suggestion
            if value == ".." or value == "":
                dates.append(value)
                continue
```

I don't know if this is a regular thing to do but I tend to interpret the branches as different behaviors that should occur. Naturally feel free to take or leave this one

Reply:
Seems sensible to me!